system/hooks/rubocop_pre_commit.rb

```diff
@@ -3,12 +3,12 @@
 require 'english'
 require 'rubocop'
 
-ADDED_OR_MODIFIED = /A|AM|^M/.freeze
+ADDED_OR_MODIFIED = /A|AM|^M/
 FILE_EXTENSIONS_TO_CONSIDER = ['.rb', '.rake'].freeze
 
 changed_files = `git status --porcelain`.split(/\n/)
                                         .select { |file_name_with_status| file_name_with_status =~ ADDED_OR_MODIFIED }
-                                        .map { |file_name_with_status| file_name_with_status.split(' ')[1] }
+                                        .map { |file_name_with_status| file_name_with_status.split[1] }
```

Comment:
Due to [Style/RedundantArgument](https://docs.rubocop.org/rubocop/cops_style.html#styleredundantargument).